cheevos-new/cheevos.c

```diff
@@ -1640,7 +1641,7 @@ static int rcheevos_iterate(rcheevos_coro_t* coro)
             if (exe_name)
             {
                scan = exe_name;
-               while (*scan != '\n' && *scan != ';' && *scan != ' ')
+               while (*scan != '\n' && *scan != '\r' && *scan != ';' && *scan != ' ')
```

Comment:
this fixes the issue where the BOOT.CNF references a file without using the ";1" suffix